### PR TITLE
Remove BoltsTest

### PR DESCRIPTION
**Pull request by grantland**
`androidTest` has been moved to `Bolts`

### CODE REVIEW

**Comment by JosephEarl** [comment]
LGTM

File: Bolts/build.gradle [hunk 1]
```diff
@@ -26,6 +26,8 @@ android {
 
 dependencies {
     testCompile 'junit:junit:4.12'
+
+    androidTestCompile 'com.android.support:support-v4:22.2.1'
```
Comment (wangmengyan95) [inline Bolts/build.gradle]: The latest version is `com.android.support:support-v4:23.0.1`, any reason we do not use that?

**Comment by wangmengyan95** [comment]
LGTM, one question.

File: Bolts/build.gradle (hunk 1 above)
Comment (wangmengyan95) [inline Bolts/build.gradle]: er... nvm, I did not see you do it in another PR.
